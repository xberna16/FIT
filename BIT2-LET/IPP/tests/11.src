.IPPcode18
DEFVAR GF@A
DEFVAR GF@B
DEFVAR GF@Vysledek
MOVE GF@A int@10
MOVE GF@B int@10
WRITE string@A=10_B=10
LT GF@Vysledek GF@A GF@B
WRITE string@LT
WRITE GF@Vysledek
GT GF@Vysledek GF@A GF@B
WRITE string@GT
WRITE GF@Vysledek
EQ GF@Vysledek GF@A GF@B
WRITE string@EQ
WRITE GF@Vysledek
