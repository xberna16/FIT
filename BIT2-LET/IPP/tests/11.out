A=10_B=10
LT
false
GT
false
EQ
true
